src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin: 10px 0;
+  }
+
+  &__logo-link {
+    line-height: 0;
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__nav-link {
+    display: flex;
+    text-decoration: none;
+    line-height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+    position: relative;
+    transition: color 0.3s;
```

Comment:
Would be nice to create a mixin for transition which will take property as a parameter, or at least a variable for duration